macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -14,39 +14,77 @@ class TestButtonViewController: NSViewController {
 			NSTextField(labelWithString: "Primary Filled"),
 			NSTextField(labelWithString: "Primary Outline"),
 			NSTextField(labelWithString: "Borderless"),
+			NSTextField(labelWithString: "Custom Button"),
 		]
-
+		
+		let customButton = Button(title: "Custom Button")
+		customButton.restBackgroundColor = Colors.Palette.green20.color
+		customButton.contentTintColor = .white
+		
 		let buttonsWithTitle: () -> [NSButton] = {
 			return [
 				Button(title: "FluentUI Button", style: .primaryFilled),
 				Button(title: "FluentUI Button", style: .primaryOutline),
 				Button(title: "FluentUI Button", style: .borderless),
+				customButton,
 			]
 		}
 
+		let customButtonWithImage = Button()
+		customButtonWithImage.restBackgroundColor = Colors.Palette.communicationBlue.color
+		customButtonWithImage.image = NSImage(named: NSImage.stopProgressTemplateName)!
+		customButtonWithImage.contentTintColor = .white
+		
 		let buttonsWithImage: () -> [NSButton] = {
 			return [
 				Button(image: NSImage(named: NSImage.stopProgressTemplateName)!, style: .primaryFilled),
 				Button(image: NSImage(named: NSImage.stopProgressTemplateName)!, style: .primaryOutline),
 				Button(image: NSImage(named: NSImage.stopProgressTemplateName)!, style: .borderless),
+				customButtonWithImage,
 			]
 		}
 		
 		let leadingArrowImage = NSImage(named: TestButtonViewController.leadingArrow)!
 		let trailingArrowImage = NSImage(named: TestButtonViewController.trailingArrow)!
 		
+		let customButtonWithTitleAndImage = Button()
+		customButtonWithTitleAndImage.title = "BtnTitle+Img"
+		customButtonWithTitleAndImage.contentTintColor = .white
+		customButtonWithTitleAndImage.image = trailingArrowImage
+		customButtonWithTitleAndImage.imagePosition = .imageTrailing
+		customButtonWithTitleAndImage.restBackgroundColor = Colors.Palette.blueMagenta30.color
 		
 		let buttonsWithTitleAndImage: () -> [NSButton] = {
 			return [
 				Button(title: "Back", image: leadingArrowImage, imagePosition: .imageLeading, style: .primaryFilled),
 				Button(title: "Skip", image: trailingArrowImage, imagePosition: .imageTrailing, style: .primaryOutline),
 				Button(title: "Back", image: leadingArrowImage, imagePosition: .imageLeading, style: .borderless),
+				customButtonWithTitleAndImage,
 			]
 		}
 		
-		let disabledButtonsWithTitleAndImage = buttonsWithTitleAndImage().map { button -> NSButton in
-			button.isEnabled = false
-			return button
+		let customDisabledButton = Button()
+		customDisabledButton.title = "DisabledWithImg"
```

Comment:
Could set the title, image, and imagePosition with a different initializer as well